tests/profiles/iso15008_test.py

```diff
@@ -0,0 +1,37 @@
+from fontbakery.codetesting import (
+    assert_PASS,
+    assert_results_contain,
+    CheckTester,
+    TEST_FILE,
+)
+from fontbakery.checkrunner import FAIL
+from fontbakery.profiles import iso15008
+
+from fontTools.ttLib import TTFont
+
+
+def test_check_dsig():
```

Comment:
"`test_check_`**`dsig`**" should be "`test_check_`**`iso15008_proportions`**"?

Reply:
D'oh.